internal/text/text.go

```diff
@@ -0,0 +1,74 @@
+package text
+
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/cli/go-gh/pkg/text"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+var whitespaceRE = regexp.MustCompile(`\s+`)
+
+func Indent(s, indent string) string {
+	return text.Indent(s, indent)
+}
+
+// Title returns a copy of the string s with all Unicode letters that begin words mapped to their Unicode title case.
+func Title(s string) string {
+	c := cases.Title(language.English)
+	return c.String(s)
+}
+
+// RemoveExcessiveWhitespace returns a copy of the string s with excessive whitespace removed.
+func RemoveExcessiveWhitespace(s string) string {
```

Comment:
This used to be called `ReplaceExcessiveWhitespace` but `RemoveExcessiveWhitespace` seemed more descriptive.